### PR TITLE
support env variables for docker

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
screenshot from 2016-01-20 15 59 20

@calston @smn ready for review
This took me way longer than I had expected, but it's working now.

**Comment by calston** [comment]
Looks okay to me :+1:

**Comment by miltontony** [comment]
Thanks! Mostly trying to tame the django beast

File: mc2/controllers/base/forms.py [hunk 1]
```diff
@@ -23,3 +24,51 @@ class Meta:
         fields = (
             'name', 'marathon_cpus', 'marathon_mem', 'marathon_instances',
             'marathon_cmd')
+
+
+class EnvVariableForm(forms.ModelForm):
+    key = forms.RegexField(
+        "^[0-9a-zA-Z_]+$", required=True, error_messages={
```
Comment (smn) [inline mc2/controllers/base/forms.py]: I'm not sure if environment variables are allowed to start with integers?

**Pull request merged**